d8/modules/custom/dmaps/dmaps.module

```diff
@@ -40,3 +40,21 @@ function dmaps_help($route_name, \Drupal\Core\Routing\RouteMatchInterface $route
       return $output;
   }
 }
+
+/**
+ * Implements hook_entity_extra_field_info().
+ */
+function dmaps_entity_extra_field_info() {
+  $info = array();
+  foreach (node_type_get_types() as $bundle) {
```

Comment:
Please use NodeType::loadMultiple() instead of node_type_get_types(), because node_type_get_types() is deprecated function